### DISCUSSION TITLE
news/20241128/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#427] [Website Discussions]
# news/20241128/

Jumper is having a Black Friday sale!

https://fcp.cafe/news/20241128/

**Comment by evrApps** [comment]
Don't forget about evrExpanse this Black Friday! 🎉

We're offering an exclusive 35% discount on evrExpanse 5 licenses! 🚀

✅ Explore enhanced features like FCP Metadata Integration, Smart LUT Finder, Burn-in Metadata, and more.

📅 Offer ends 8th December 2024 – use code EVRBF2024 at checkout!

Learn more and grab your license here: https://www.evrapp.cloud/evrexpanse

Happy savings! 🛍️✨
